### DISCUSSION TITLE
哪个可以符合例程要求的ESP32开发板

### DISCUSSION THREAD

**Discussion by jaybajing** [espressif/esp-csi#215] [General]
为了保证 CSl 的感知效果，请尽量满足以下要求:
1.Use ESP32-C3/ESP32-S3: ESP32-C3/ESP32-S3 is the best RF chip at present
2. Use an external antenna: PCB antenna has poor directivity and is easily interfered by the motherboard
3. The distance between the two devices is more than one meter

**Comment by NingYuxiang** [comment]
我们目前暂未提供ESP32-C3和ESP32-S3具有外接天线的开发板，但是我们提供具有外接天线的ESP32-C3、ESP32-S3的模组[ESP32-S3-WROOM-1U](https://www.espressif.com/sites/default/files/documentation/esp32-s3-wroom-1_wroom-1u_datasheet_cn.pdf)、[ESP32-S3-MINI-1U](https://www.espressif.com/sites/default/files/documentation/esp32-s3-mini-1_mini-1u_datasheet_cn.pdf)、[ESP32-C3-MINI-1U](https://www.espressif.com/sites/default/files/documentation/esp32-c3-mini-1_datasheet_cn.pdf)、[ESP32-C3-WROOM-02U](https://www.espressif.com/sites/default/files/documentation/esp32-c3-wroom-02_datasheet_cn.pdf)。您可以自己绘制PCB板焊接外接天线模组，或者购买[ESP32-S3-DevKitC-1](https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1.html)、[ESP32-S3-DevKitM-1](https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitm-1.html)、[ESP32-C3-DevKitM-1](https://docs.espressif.com/projects/espressif-esp-dev-kits/zh_CN/latest/esp32c3/esp32-c3-devkitm-1/index.html)等开发板，使用外接天线模组替换原有模组使用。
此外，自带的 PCB 天线在 CSI 应用中的表现也非常出色，您也可以尝试使用PCB天线的开发板进行体验。
